src/validators/fieldset-validators/fieldset-validators.ts

```diff
@@ -0,0 +1,70 @@
+import { Validator } from "../validator";
+
+export const requiredFieldset: Validator<string> = {
+    validate: (id: string) => {
+        const el = document.querySelector(`gcds-fieldset[fieldset-id=${id}]`);
+        const elChildren = el.children;
+
+        let isValid = validateFieldsetElements(el, elChildren);
+
+        return !isValid.includes(false);
+    },
+    errorMessage: { "en": "Please complete the required field to continue.",  "fr": "Veuillez compléter les champs obligatoires afin de continuer." }
+}
+
+export function validateFieldsetElements(element, nodeList) {
+    let isValid = [];
+
+    for (var i = 0; i < nodeList.length; i++) {
+        switch(nodeList[i].nodeName) {
+            case('GCDS-FIELDSET'):
+                let validfieldsetChildren = validateFieldsetElements(nodeList[i], nodeList[i].children);
```

Comment:
Very small change but should the f of fieldset in validFielsetChildren be capitalized?